test/mocks/http/alternate_protocols_cache.cc

```diff
@@ -0,0 +1,13 @@
+#include "test/mocks/http/alternate_protocols_cache.h"
+
+namespace Envoy {
+namespace Http {
+
+MockAlternateProtocolsCache::~MockAlternateProtocolsCache() = default;
```

Comment:
At this point I'd just define all this in the header file, but up to you. I don't know what the rest of Envoy does in this scenario.

Reply:
Yeah, I share your intuition. I followed the pattern for the other files in test/mock/http and put the = default in the .cc file.